arduino-ide-extension/src/browser/theia/core/application-shell.ts

```diff
@@ -31,6 +32,18 @@ export class ApplicationShell extends TheiaApplicationShell {
   @inject(ConnectionStatusService)
   protected readonly connectionStatusService: ConnectionStatusService;
 
+  async setLayoutData(
+    layoutData: TheiaApplicationShell.LayoutData
+  ): Promise<void> {
+    layoutData.activeWidgetId = SketchbookWidget.ID;
+    layoutData.leftPanel?.items?.forEach((item) => {
+      if (item?.widget?.id === SketchbookWidget.ID) item.expanded = true;
+      else item.expanded = false;
+    });
```

Comment:
Shouldn't just `layoutData.activeWidgetId = SketchbookWidget.ID;` be enough? It automatically expands the selected widgets, thereby marking all the other widgets in the left panel not expanded.

Reply:
Oh, right! I thought that wasn't enough because it felt weird to leave `expanded` set to false but it seems to work. Thanks!

Reply:
Wait, forget what I said. It doesn't work when you select another widget from the sidebar and then reopen the sketch.

Reply:
Interesting, good to know. Then feel free to ignore my comment :)